packages/server/lib/plugins/dev-server.js

```diff
@@ -6,11 +6,22 @@ const plugins = require('../plugins')
 
 const baseEmitter = new EE()
 
+let allowToRequest = false
```

Comment:
"canRequest" is a bit better "shouldRequest" is also ok

Reply:
this is not needed, removing right now

Reply:
I'm not a fan of using the module scope to init the state... but I understand that the existing model requires plugins.registerHandler to be executed when the file is imported